README.md

```diff
@@ -38,7 +38,7 @@ Bevy is still in the _very_ early stages of development. APIs can and will chang
 
 ## Community
 
-Before contributing or participating in discussions with the community, you should familiarize yourself with our **[Code of Conduct](./CODE_OF_CONDUCT.md).
+Before contributing or participating in discussions with the community, you should familiarize yourself with our [**Code of Conduct**](./CODE_OF_CONDUCT.md).
```

Comment:
Should this be inside the link or around the link?

Reply:
Functionally, they are the same. Semantically, I like the current phrasing better. "I want this bold text to be a link" makes more sense to me than "I want this link to be bold".